interfaces/Portalicious/src/app/pages/projects-overview/project-metric-container/project-metric-container.component.ts

```diff
@@ -0,0 +1,19 @@
+import { getRandomInt } from '@121-service/src/utils/getRandomValue.helper';
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { SkeletonModule } from 'primeng/skeleton';
+
+@Component({
+  selector: 'app-project-metric-container',
+  standalone: true,
+  imports: [SkeletonModule],
+  templateUrl: './project-metric-container.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProjectMetricContainerComponent {
+  public status = input.required<string>();
```

Comment:
```suggestion
  public loading = input.required<boolean>();
```

I think this is a bit more explicit